openedx/adg/lms/applications/models.py

```diff
@@ -30,12 +35,43 @@ def __str__(self):
         return 'User {user_id}, application status id={id}'.format(user_id=self.user.id, id=self.id)
 
 
+class BusinessLine(TimeStampedModel):
```

Comment:
Please add verbose_name in all the fields of updated models to make field names localized when used in model forms.

Reply:
verbose_name are used if you want to make your model attribute more readable, there is no need to define verbose name if your field attribute is easily understandable. Django automatically create it using field name.

Reply:
Yes, django automatically create it using the field name by removing the underscores but it will not localize the field name. So to make it translatable we can add verbose_name. So that if user select Arabic language the field names ll also be translated.